### ISSUE TITLE
"Alt l" doesn't work as expected inside Zellij

### ISSUE THREAD

**Issue by rempas** [zellij-org/zellij#2908]
**Basic information**

`zellij --version`: zellij 0.38.2
`stty size`: WHAT'S THAT???
`uname -av` or `ver`(Windows): Linux Arco 6.5.9-arch2-1 #1 SMP PREEMPT_DYNAMIC Thu, 26 Oct 2023 00:52:20 +0000 x86_64 GNU/Linux

List of programs you interact with as, `PROGRAM --version`: output cropped meaningful, for example:
`zsh 5.9 (x86_64-pc-linux-gnu)`

**Further information**
First, I wanted to say that I found Zellij 2 days ago, and I think that it's super awesome!!! I was finally able to customize it to 100% of my liking (I don't have lots of needs) after reading the docs, but I have only one problem.

I am using Foot and ZSH and I have bind "Alt l" to go to the previous line in history (`bindkey '^[[A' history-substring-search-up`) and "Alt t" to go to the next line in history (`bindkey '^[[B' history-substring-search-down`).

"Alt t" seems to be working without problems, but "Alt l" doesn't work. I have also tried to unbind both of them globally in my `config.kdl` but that didn't work. I can also post my whole `.zshrc` if needed.

**Comment by tgross35** [comment]
Are you able to jump around words? I am having a similar problem in Zsh specifically, https://github.com/zellij-org/zellij/discussions/3261. Bash, Nushell and other shells seem to work fine, as does Zsh when not interfacing with Zellij, but Zellij with Zsh breaks navigation with alt for some reason.

I am on MacOs.

**Comment by rempas** [comment]
> Are you able to jump around words? I am having a similar problem in Zsh specifically, #3261. Bash, Nushell and other shells seem to work fine, as does Zsh when not interfacing with Zellij, but Zellij with Zsh breaks navigation with alt for some reason.
> 
> I am on MacOs.

I don't use neither Zellij (and don't have the config files to try) neither the "history-substring" plugin so I cannot try. I can however try to remember (even tho lots of months have passed). What do you mean with "jump around words"?

**Comment by tgross35** [comment]
Just alt+left or alt+right to move the cursor by a whole word, which I think does the same as alt+b and alt+f. No worries if you don't know of the top of your head, it seems likely to come from the same issue.

**Comment by rempas** [comment]
> Just alt+left or alt+right to move the cursor by a whole word, which I think does the same as alt+b and alt+f. No worries if you don't know of the top of your head, it seems likely to come from the same issue.

Yeah, I suppose that it does. I am using VI mode so, the bindings are mapped to what I want them. I unfortunately haven't tried another shell.
